client/src/app/components/forms/rsvp/rsvp.component.scss

```diff
@@ -1,4 +1,43 @@
+@import '../../../../styles.scss';
+
+hr {
+	//TODO: Figure out how to pull in ll-green 800 :/
```

Comment:
Does importing wedding-theme.scss fail?

Reply:
it's much more...weird. technically, no - you have to be pretty picky about it.

but you can't actually get the colors from there! for some fucking reason? it just throws several hissy fits no matter how you go about it - accessing the color from within the palette fails.

Reply:
Oof, we can look into that later then